Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -381,6 +381,45 @@ Needle contains a module aimed to bypass non-specific jailbreak detection implem
 [needle][script_jailbreak-detection-bypass] > run
 ```
 
+
+
+#### Using getppid
```

Comment:
General question: does this work on release builds? It'd be nice if you could name some apps know to be implementing this.

Reply:
Sure, it works on Release builds from XCODE and also on apps from AppStore. You can easily verify it by using iOS-deploy and then 'ps -efj'. Have a look at Airbnb and Goodreads but they don't crash on it. They seem to send this data to metric services. I found the use of _getppid_ on 151 apps from 1300.